examples/spire/README.md

```diff
@@ -86,12 +84,12 @@ WEST_GATEWAY_IP=$(kwest get svc federation-ingress-gateway -n istio-system -o js
 helm-east install east-mesh chart -n istio-system \
     --values examples/kind/east-federation-controller.yaml \
     --set "istio.spire.enabled=true" \
-    --set "federation.meshPeers.remote.addresses[0]=$WEST_GATEWAY_IP"
+    --set "federation.meshPeers.remote.addresses[0]=$WEST_GATEWAY_IP" \
```

Comment:
This backslash breaks this command.
```suggestion
    --set "federation.meshPeers.remote.addresses[0]=$WEST_GATEWAY_IP"
```

Reply:
Humm weird. I tested it yesterday and might have missed it.